src/pages/EIP/List/index.jsx

```diff
@@ -57,14 +56,14 @@ class List extends Component {
             <Icon name='eip-duotone' size={40} />
             <div className={styles.nametexts}>
               <div className={styles.name}>
-                <text>{name}</text>
+                <p>{name}</p>
                 {isDefault ? <div>
-                  <text>{isDefault}</text>
+                  <p>{isDefault}</p>
                 </div> : ''}
```

Comment:
In react, multi-line html elements, wrapped in parentheses are better. Eg: 
```
(
   <div>
       <span>Text<span>
   <div>
)
```

Reply:
Yes, but you can modify only the parts of this pr. We can later modify this type of issue specifically.

Reply:
Cool.